### PR TITLE
Milestone 4 : Add Controlles and Views

### PR DESCRIPTION
**Pull request by ABDULALI3468**
> ## What I did ♻️
> > In this project's milestone I create all `controllers` and `views` for my [Blog app](https://github.com/Chrissiku/Rails-Blog). As II won't be processing any data yet, I add simple placeholders for my `views` with plain HTML.
> 
> ## Project requirements 📗
> * 🔴  **Important:** do NOT use `rails generate scaffold` command. The point of this project is to show you how to set up controllers by yourself.
> * For each URL you should create:
>   
>   * A route.
>   * An action in the correct controller.
>   * A view file.
> * View files should include only placeholders written in plain HTML, e.g:
>   
>   * Here is a list of posts for a given user
> * Make sure that you added code to handle **only** the 4 required URLs.
> 
> ## Show your support
> Give a ⭐️ if you like this project!

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class PostsController < ApplicationController
+  def index; end
 end
```
Comment (brytebee) [inline app/controllers/posts_controller.rb]: - [ ] **Required:** Kindly include the `show` action in the `PostsController`.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ABDULALI3468👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design.✔️
- App functions as it should.✔️
- The `README` looks professional.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ABDULALI3468,

✔️ Implementation of routes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ABDULALI3468** [comment]
Thanks, guys for giving me a good review. @brytebee  @TedLivist

**Pull request merged**